src/wired/dataclasses/injector.py

```diff
@@ -19,7 +19,7 @@ class Injector:
 
     def __call__(self, container):
         target = self.target
-        context = container.context
+        context = getattr(container, 'context', None)
```

Comment:
are there scenarios right now where this matters? It looks like `container.context` is reliable right now via https://github.com/mmerickel/wired/blob/5ee2703a7470489e99badba12a555fbf1e5cc59d/src/wired/container.py#L137

Reply:
Hmm, you're right, per the container code, it shouldn't be possible to get into that situation. I will remove that change and see if maybe it was a dummy/mock of mine that caused that.

Reply:
Pushed with that switched back.